proxy/yeager/client.go

```diff
@@ -121,24 +120,29 @@ func (c *Client) DialContext(ctx context.Context, addr string) (net.Conn, error)
 }
 
 const (
-	addressIPv4   = 0x01
-	addressDomain = 0x03
+	networkTCP = 0x01
+	networkUDP = 0x02
+)
```

Comment:
would be better with comment:
// the value defined for NETWORK in yeager protocol